libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -408,6 +408,27 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     }
 }
 
+.gd-invite-button {
```

Comment:
Please update the copyright section of this file as well.

Reply:
I updated it thanks